Corgibytes.Freshli.Cli/CommandOptions/CheckoutHistoryCommandOptions.cs

```diff
@@ -4,6 +4,6 @@ public class CheckoutHistoryCommandOptions : CommandOptions
 {
     public string RepositoryId { get; set; }
     public string Sha { get; set; }
-    public string GitPath { get; set; }
+    public string GitPath { get; set; } = "git";
```

Comment:
We do NOT want to provide this here. `System.CommandLine` should be providing the default value, which is similarly defined in `Commands/Git/GitCloneCommand.cs:20`. This is necessary for the `--help` documentation to inform the user of the default, and of course we don't want to define twice.